README.md

```diff
@@ -1,24 +1,52 @@
-# README
+# Recipe
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+-  Ruby on Rails
+-  PostgreSQL
 
-* System dependencies
+## Live Demo
+[Live Demo](https://mustbudgetapp.herokuapp.com/)
```

Comment:
-  [ ] Great job for deploying your project to GitHub. However, the attempt to sign up using the **online demo app is unsuccessful**. Nothing happens when new user details are entered and the _sign up_ button is clicked. Kindly ensure that the online demo is working correctly.